test/system/220-healthcheck.bats

```diff
@@ -105,6 +105,46 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\"
     run_podman rmi $img
 }
 
+@test "podman wait --condition=healthy" {
+    ctr="healthcheck_c"
+    img="healthcheck_i"
+    wait_file="$PODMAN_TMPDIR/$(random_string).wait_for_me"
+    _build_health_check_image $img
+
+    for condition in healthy;do
```

Comment:
Missing "unhealthy" here; not sure if that was deliberate (for CI testing?) or accidental.

Reply:
Accident. Meetings really throw me back to 0 when working on a Pr. Thanks for catching!